code/modules/clothing/masks/gasmask.dm

```diff
@@ -64,6 +111,7 @@
 	icon_state = "plaguedoctor"
 	inhand_icon_state = "gas_mask"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 2,ENERGY = 2, BOMB = 0, BIO = 75, RAD = 0, FIRE = 0, ACID = 0)
+	tint = 0
```

Comment:
I don't get why you're going to keep showing this preference to the plague doctor mask, but not to any of the other costume gasmask children. Like, plaguedoctor masks are super obfuscating visually, but you've decided that the owl mask, tiki mask, carp mask, bounty hunter mask, and cyborg visor ***should*** have a tint?

Thanks for minding the performer job masks.

Reply:
> Aye, managed that, now when wearing a gasmask the screen darkens (except for mime and clown masks)

This makes your choice to save plague doctor masks even more confusing. Is this like a mask that you enjoy wearing and plan to be making use of? Please show the other costume masks the same level of care that you are giving this one.

Reply:
oh it's a tint i missed when i was doing code, first i had it like this for all masks, but then i changed things around and forgot about this, i'll remove the tint=0 from plague masks